### PR TITLE
feature/M0127CC-611 JSON article/tutorial

### PR DESCRIPTION
**Pull request by fpinel-microej**

### CODE REVIEW

File: Tutorials/tutorialJsonParser.rst [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+The example will parse the file, browse the resulting data structure (``org.json.me.JSONObject``) and print the value of the ``menuitem`` JSON array.
+
+.. code:: Java
+
+	/*
+	 * Java
+	 *
+	 * Copyright  2014-2020 MicroEJ Corp. All rights reserved.
+	 * Use of this source code is governed by a BSD-style license that can be found at http://www.is2t.com/open-source-bsd-license/.
```
Comment (fpinel-microej) [inline Tutorials/tutorialJsonParser.rst:54]: Do we have a more up-to-date in-house link to the BSD License ?

File: Tutorials/tutorialJsonParser.rst [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+	 * MicroEJ Corp. PROPRIETARY. Use is subject to license terms.
```
Comment (fpinel-microej) [inline Tutorials/tutorialJsonParser.rst:55]: I kinda copy-pasted this but is this code snippet Proprietary ? The license in the original article didn't mention this

**Review by fpinel-microej — COMMENTED** [review]
Should we talk about how to include the menu.json resource to run as a MicroEJ Java project ?

File: Tutorials/tutorialJsonParser.rst [hunk 3]
```diff
@@ -0,0 +1,145 @@
+JSON parser on MicroEJ
```
Comment (stephanemainchain) [inline Tutorials/tutorialJsonParser.rst:1]: Using a JSON parser

File: Tutorials/tutorialJsonParser.rst [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+This article explains how to parse and use JSON files with MicroEJ.
+
+JSON and its adapted parser
+---------------------------
+
+As described on the `JSON official site <http://json.org/>`_, JSON (JavaScript Object Notation) is a lightweight data-interchange format. It is widely used in many fields for example as a mean of data serialization in lightweight web services such as REST, for server interrogation in Ajax for dynamic webpage building, or even databases.
+
+Due to its lightweight nature and its simplified syntax compared to XML, it is quite natural to use it in embedded systems. JSON is easily readable by humans but what about computers? That is where the official JSON page comes in handy by providing a rather impressive list of JSON parsers. For our convenience a Java mobile edition (ME) of a JSON parser is available.
+
+Parser use in MicroEJ
```
Comment (stephanemainchain) [inline Tutorials/tutorialJsonParser.rst:13]: Parser use in MicroEJ SDK

File: Tutorials/tutorialJsonParser.rst (hunk 1 above)
Comment (frivieremicroej) [reply Tutorials/tutorialJsonParser.rst:54]:
```
/*
 * <Language>
 *
 * Copyright <beginYYYY>-<LastModifYYYY> MicroEJ Corp. All rights reserved.
 * Use of this source code is governed by a BSD-style license that can be found with this software.
 */
```

**Pull request closed without merging**